db/migrate/20160106224158_create_stories.rb

```diff
@@ -4,6 +4,9 @@ def change
       t.string :uid
       t.string :text
       t.string :url
+      t.string :media_content
+      t.integer :subscription_id
+      t.datetime :post_time
```

Comment:
When you're adding additional columns to a table, use a migration instead of manually changing the existing migration.